cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/increment/hbase96/IncrementSummingScanner.java

```diff
@@ -221,14 +234,18 @@ private boolean nextInternal(List<Cell> cells, int limit) throws IOException {
     // emit any left over increment, if we hit the end
     if (previousIncrement != null) {
       // in any situation where we exited due to limit, previousIncrement should already be null
-      Preconditions.checkState(limit <= 0 || addedCnt < limit);
+      Preconditions.checkState(limit <= 0 || addedCnt < limit, "addedCnt={}, limit={}", addedCnt, limit);
```

Comment:
The format specifier for Preconditions.checkState is ``%s`` - http://guava-libraries.googlecode.com/svn-history/r21/trunk/javadoc/com/google/common/base/Preconditions.html#checkState(boolean, java.lang.String, java.lang.Object...)